Tests/SonarQube.MSBuild.Tasks.IntegrationTests/SonarQube.MSBuild.Tasks.IntegrationTests.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SonarQube.MSBuild.Tasks.IntegrationTests</RootNamespace>
     <AssemblyName>SonarQube.Integration.Tasks.IntegrationTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
@duncanpMS I've put on our documentation page that the minimum required .NET version we need is 4.5.2 - do you think that it is correct?